src/camera-controls.js

```diff
@@ -291,15 +291,15 @@ export default class CameraControls extends EventDispatcher {
 				if ( scope._state === STATE.TOUCH_DOLLY_TRUCK ) {
 
 					// 2 finger pinch
-					const dx = _v2.x - event.touches[ 1 ].pageX;
```

Comment:
Is there any reason we should use `pageX` / `pageY` instead of `clientX` / `clientY` at here?

Reply:
All changes belongs to the PR are just replacing these `pageX` / `pageY` into `clientX` / `clientY`

Reply:
Good finding. you are right.
I have not tested with laid out canvas elements...
thanks!